regions_www/m/server.R

```diff
@@ -19,804 +19,1071 @@
-    )
-  })
-
-  output$route_codebook <- renderUI({
-    a("Codebook", href = paste(
-      "https://cdn.rawgit.com/npct/pct-shiny", repo_sha, "static", "codebook_routes.csv", sep = "/"),
-      title="This explains the variable names in the downloadable data",
-      onclick="ga('send', 'event', 'download', 'route_codebook');", target='_blank'
+  ## Updates map tile according to the selected map base
+  map_tile <- reactive({
+    switch(
+      input$map_base,
+      'roadmap' = list(url = "https://cartodb-basemaps-{s}.global.ssl.fastly.net/light_all/{z}/{x}/{y}.png", zoom = 18),
+      'satellite' = list(url = "https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}", zoom = 18),
+      'IMD' = list(url = "https://maps.cdrc.ac.uk/tiles/imd2015_eng/{z}/{x}/{y}.png", zoom = 14),
+      'opencyclemap' = list(url = "https://{s}.tile.thunderforest.com/cycle/{z}/{x}/{y}.png?apikey=feae177da543411c9efa64160305212d", zoom = 18),
+      'hilliness' = list(url = "https://server.arcgisonline.com/ArcGIS/rest/services/World_Shaded_Relief/MapServer/tile/{z}/{y}/{x}", zoom = 13)
     )
   })
 
-  output$route_codebook_quiet <- renderUI({
-    a("Codebook", href = paste(
-      "https://cdn.rawgit.com/npct/pct-shiny", repo_sha, "static", "codebook_routes.csv", sep = "/"),
-      title="This explains the variable names in the downloadable data",
-      onclick="ga('send', 'event', 'download', 'route_codebook');", target='_blank'
-    )
-  })
 
-  output$route_network_codebook <- renderUI({
-    a("Codebook", href = paste(
-      "https://cdn.rawgit.com/npct/pct-shiny", repo_sha, "static", "codebook_rnet.csv", sep = "/"),
-      title="This explains the variable names in the downloadable data",
-      onclick="ga('send', 'event', 'download', 'route_network_codebook');", target='_blank'
-    )
-  })
+  ##############
+  # Map attribute outputs
+  ##############
 
-  output$zone_codebook <- renderUI({
-    a("Codebook", href = paste(
-      "https://cdn.rawgit.com/npct/pct-shiny", repo_sha, "static", "codebook_zones.csv", sep = "/"),
-      title="This explains the variable names in the downloadable data",
-      onclick="ga('send', 'event', 'download', 'zones_codebook');", target='_blank'
-    )
-  })
-
-  # Initialize the leaflet map
+  ## Initialize the leaflet map
   output$map <- renderLeaflet(
     leaflet() %>%
-      addCircleMarkers(., data = to_plot$cents, radius = 0, group = "centres", opacity = 0.0) %>%
-      setView(., lng = to_plot$center_dim[1, 1], lat = to_plot$center_dim[1, 2], zoom = 10) %>%
+      # Centroids loaded invisibly to tell it the extent of the map
+      # Anna note: centroids do not exist for some purposes like schools - if ever want to *start* in schools layer need to change this to zones
+      # addCircleMarkers( .,
+      #   data = to_plot$centroids,
+      #   radius = 0,
+      #   group = "centroids",
+      #   opacity = 0.0
+      # ) %>%
```

Comment:
Is there a reason for this commented out code staying?

Reply:
No.

Reply:
:cool: I'll delete it then

Reply:
yes fine – it was there in our previous pct-shiny version and the comment "Centroids loaded invisibly to tell it the extent of the map" made it sound like it was doing something – but I agree it does not in fact seem to be needed